pages/components/avatar/avatar.css

```diff
@@ -0,0 +1,62 @@
+@import url("/utils/color.css");
+.sui_avatar {
```

Comment:
naming convention suggests using hyphen(-) for multi word class names, eg: sui-avatar instead of sui_avatar